config.py

```diff
@@ -71,6 +71,11 @@ class Config(object):
     DM_APP_NAME = 'buyer-frontend'
     DM_DOWNSTREAM_REQUEST_ID_HEADER = 'X-Amz-Cf-Id'
 
+    DM_FRAMEWORK_CONTENT_MAP = {
```

Comment:
I like the way putting this into config gives the impression that we don't have hard-coded framework slugs in our frontend apps while still having hard-coded framework slugs in our apps :stuck_out_tongue: 

I think it's worth a short comment in the config file to explain what this is used for, because it's pretty esoteric otherwise.